src/components/pages/TaskType.vue

```diff
@@ -639,6 +645,7 @@ export default {
       const departments = this.user.departments || []
       return (
         this.isCurrentUserManager ||
+        !departments.length ||
```

Comment:
Rule to fix:

`this.isCurrentUserSupervisor && ( !departments.length || departments.includes(...) )`